src/pages/getting-started/livechat-apps/index.mdx

```diff
@@ -46,6 +42,129 @@ See a 4-minute tutorial explaining how to create a simple app in the Developer C
   allowfullscreen
 ></iframe>
 
+
+## Step-by-step guide
+
+If you want to know the whole creation process in detail, follow the instructions we’ve prepared. We'll use an app named **Your Guide** as our example.
+
+### **Step 1 Getting started**
+
+Make sure you are signed in to the [Developer Console](https://developers.livechat.com/console/). Your LiveChat credentials should allow you to be logged in automatically. If not, you can [sign up here](https://accounts.livechat.com/signup).
+
+Now, we go to the **Apps** tab and click the **Create new app +** button. Give your app a name and click the **Continue** button. You can always change the application's name later.
+
+💡 **Need a helping hand?** Right after signing in to the Developer Console, you can launch the **App wizard**. It will guide you through the whole app creation process.
+
+<img
+  alt="Creating LiveChat apps step 1, App wizard"
+  src="/images/getting-started/livechat-apps/LiveChat-apps-step1.png"
+  width="1515px"
+  height="819px"
+/>
+
+### **Step 2 Adding Agent App Widgets block**
+
+You can configure your app using Building blocks, which we can describe as components that form your app. Depending on which building block you add to your application, it will work differently. If you want to know how each building block can enrich your app, click the **Read docs** redirect and learn more about them. 
+
+To create a simple frontend app, we'll use the **Agent App Widgets building block**. 
+Agent App Widgets are web apps that can interact and communicate with the LiveChat App, loaded inside the LiveChat App interface.
+
+<img
+  alt="Creating LiveChat apps step 2, adding Agent App Widgets block"
+  src="/images/getting-started/livechat-apps/LiveChat-apps-step2.png"
+  width="1500px"
+  height="850px"
+/>
+
+To add a new Agent App Widget to your application, you need to follow these steps:
+1. Go to the **Building blocks** tab. 
+2. Add a new Agent App Widgets block by clicking the **Get started** button. 
+3. Enter the **Widget source URL**, which is the address of your application. If it's not ready yet, you can use our sample apps to play around.
+4. Now, go to the **Widget placement** and choose one. It allows you to decide where exactly in the LiveChat App you want to display your integration. We've chosen the **Full screen** option in our example. In the LiveChat App, Full screen apps are located in the navigation bar on the left, and they open maximized in the LiveChat App. [Learn more](https://developers.livechat.com/docs/extending-agent-app#main-menu-fullscreen-apps)
```

Comment:
```suggestion
4. Now, go to the **Widget placement** and choose one. It allows you to decide where exactly in the LiveChat App you want to display your integration. We've chosen the **Full screen** option in our example. In the LiveChat App, Full screen apps are located in the navigation bar on the left, and they open maximized in the LiveChat App. [Learn more...](https://developers.livechat.com/docs/extending-agent-app#main-menu-fullscreen-apps)
```

Reply:
And I was thinking about these 3dots xd I will remember that :)